Database/ReleaseScript/0624 - Add project document type table and fk for projectdocument table.sql

```diff
@@ -0,0 +1,44 @@
+create table dbo.ProjectDocumentType(

```

Comment:
I think you should make this a lookup table and use the pattern where Name is no spaces and DisplayName is with spaces and userfacing. You could set it up as an enum too, but not necessary